lib/fluent/plugin/filter_process_redfishalert.rb

```diff
@@ -24,7 +27,7 @@ def filter(tag, time, record)
      begin
       # REMOTE_ADDR is the IP the event was sent from
       rmcSN = getRMCSerialNumber(record["REMOTE_ADDR"])
```

Comment:
we can add an "if" here for the hardware type and call the appropriate method for what redfish call we're trying to make

Reply:
I need some clarification on this. Are you advising removing the switching within the func getRMCSerialNumber() and getRackSerialNumber() into the filter function?